haystack/utils/squad_data.py

```diff
@@ -109,8 +109,13 @@ def to_df(data):
         """Convert a list of SQuAD document dictionaries into a pandas dataframe (each row is one annotation)"""
         flat = []
         for document in data:
-            title = document["title"]
+            title = ""
+            if "title" in document:
+                title = document["title"]
             for paragraph in document["paragraphs"]:
+                document_id = ""
```

Comment:
That would result in documents without a `document_id` in paragraph receiving the same, empty `document_id`. Is that really what we would like to achieve or could that later on lead to problems? What about using the document title to generate an id as a hash of the title, similar to what we do here: https://github.com/deepset-ai/haystack/blob/ac5617e757e9ace6f30b7291686d9dbbc339f433/haystack/schema.py#L117

Reply:
Ah, and please don't forget to add labels to the PR.

Reply:
Yes good idea, I have tried to generate document_id by hashing the actual text content since sometimes we don't have a title (e.g. data from annotation tool). Labels have also been added now!